docs/detections/alerts-view-details.asciidoc

```diff
@@ -65,8 +65,26 @@ If you have a https://www.elastic.co/pricing[Platinum or Enterprise subscription
 * *Alerts related by process ancestry* - Shows alerts that are related by process events on the same linear branch. Note that alerts generated from processes on child or related branches are not shown. To further examine alerts, click *Investigate in timeline*.
 
 [discrete]
-[[enriched-data-overview]]
-==== Enriched data on alerts
+[[alerts-enrich-host-user-risk-score]]
+==== Alerts enriched with user and host risk scores
+Alerts can be enriched with user and host risk scores, which convey the level of risk associated with a specific user and host. Risk levels can be `Unknown`, `Low`, `Moderate`, `High`, or `Critical`. Refer to <<host-risk-score>> and <<user-risk-score>> to learn how risk scores are calculated.
+
+NOTE: User and host risk scores are technical preview features. You must enable user and host risk score features to access risk scores data. Refer to <<host-risk-score>> and <<user-risk-score>> to learn more.
+
+[role="screenshot"]
+image::images/enriched-host-user-rs.png[Host and user risk score subsections with risk scores, 600]
+
+Two types of user and host risk scores can appear in the Enriched data section:
+
+* **Current user/host risk classification:** The current risk score of the user or host associated with the alert.
+* **Original user/host risk classification:** The first risk score that was calculated for the user or host associated with the alert.
++
+The *Original user/host risk classification* field only displays if the current risk score no longer matches the original risk score. In this situation, both the current and original risk scores appear, showing how the risk changed. The current risk score appears in the *Current user/host risk classification* field. The original risk score appears in the *Original user/host risk classification* field.
```

Comment:
The last two sentences seem pretty obvious, based on the field names. We also just explained this in the bullets above
```suggestion
The *Original user/host risk classification* field only displays if the current risk score no longer matches the original risk score. In this situation, both the current and original risk scores appear, showing how the risk changed.
```